.env.example

```
@@ -0,0 +1,6 @@
+VITE_IDP_CLIENT_ID=idp_client_id
+VITE_IDP_CLIENT_SECRET=idp_client_secret
+VITE_IDP_REDIRECT_URI=idp_redirect_uri
+VITE_IDP_AUTHORIZE_URL=idp_authorize_url
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_IDP_AUTHORIZE_URL key should go before the VITE_IDP_CLIENT_ID key

(UnorderedKey)